float32[] goal_state
float32 goal_radius
float32 time_limit
<<<<<<< HEAD
float32 probability_success_threshold
string planning_algorithm
=======
float32 probability_success_threshold
>>>>>>> d8bdba09fb630f5cb93340f38caebb1b23810fd9
